src/main/fc/fc_msp.c

```diff
@@ -142,13 +142,13 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXSONAR, "SONAR", 22 },
     { BOXSERVO1, "SERVO1", 23 },
     { BOXSERVO2, "SERVO2", 24 },
-    { BOXSERVO3, "SERVO3", 25 },
     { BOXBLACKBOX, "BLACKBOX", 26 },
     { BOXFAILSAFE, "FAILSAFE", 27 },
     { BOXAIRMODE, "AIR MODE", 28 },
-    { BOX3DDISABLESWITCH, "DISABLE 3D SWITCH", 29},
-    { BOXFPVANGLEMIX, "FPV ANGLE MIX", 30},
+    { BOX3DPOSSWITCH, "POSTIVE ONLY 3D", 29},
+    { BOX3DNEGSWITCH, "NEGITIVE ONLY 3D", 30},
```

Comment:
I can see the problem with accidentally going full throttle negative. My approach to fix this would be to add this to the mixer on your TX, so that the switch that activates `BOX3DDISABLESWITCH` also changes the mixer output range of your throttle channel to be limited to 1400 to 2000. I am just not sure that splitting this up into 2 switches is going to add new functionality, and not just make it more confusing to be configured.